package fibanocci;

public class trycatchfinally
{
	public static void main(String[]args)
	{
		try
		{
			int a=10;
			int b=0;
			int returns=a/b;
			System.out.println("the value is:"+returns);
		}
		catch (ArithmeticException e)
		{
			
			System.out.println("Error:"+e);
		}
		finally
		{
			System.out.println("Finally executed!!!");
		}
		
	}
}
